terraform/environments/coat/s3.tf

```
@@ -288,3 +288,29 @@ resource "aws_s3_object" "pod_waste_reports" {
   key    = "pod_waste_reports/"
   acl    = "private"
 }
+
+
+# COAT GitHub repositories Terraform state bucket
+module "coat_github_repos_tfstate_bucket" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+  count = local.is-production ? 1 : 0
+
+  source  = "terraform-aws-modules/s3-bucket/aws"
+  version = "4.3.0"
+
+  bucket = "coat-github-repos-tfstate"
```

```suggestion
  bucket = "coat-github-repos-tfstate-bucket"
```
To match name given in the TF code?